.vale.ini

```diff
@@ -0,0 +1,8 @@
+StylesPath = ../.vale/styles
+
+MinAlertLevel = suggestion
+
+Packages = https://github.com/rohennes/ocp-rules/releases/latest/download/AsciiDoc.zip
+
+[[!.]*.adoc]
+BasedOnStyles = AsciiDoc
```

Comment:
Need to include the RedHat style otherwise it won't be applied to the modules folder:
```

StylesPath = ../.vale/styles

MinAlertLevel = suggestion

Packages = RedHat, \
https://github.com/rohennes/ocp-rules/releases/latest/download/OpenShiftAsciiDocValidation.zip

[[!.]*.adoc]
BasedOnStyles = RedHat, OpenShiftAsciiDocValidation
```